app/core/components/ContributorPathReport/index.tsx

```diff
@@ -92,7 +88,20 @@ export const ContributorPathReport = ({
         );
       },
     },
-    { field: "name", headerName: "Name", flex: 1 },
+    {
+      field: "name",
+      headerName: "Name",
+      flex: 1,
+      renderCell: (cellValues: any) => {
+        console.log(cellValues.value[0]);
+
+        return (
+          <a href={`mailto:${cellValues.value[0]?.email}`}>
```

Comment:
better use our own Link component (app/core/components/Link.tsx) or MUI's, to have better colors on these links.